tenant/sunbird/index.html

```diff
@@ -146,7 +147,8 @@ <h1 class="sb-text-upper mt-0">Learning Platform</h1>
                             <h2 class="sb-text-upper">National Urban</h2>
                             <h1 class="sb-text-upper mt-0">Learning Platform</h1>
                             <p class="sb-font">एसबीएम 2.0 ‘कचरा मुक्त शहर ’ बनाने की उद्देश्य के साथ लॉन्च किया जा रहा है, <br> जिसका लक्ष्य शहरी स्वच्छता और अपशिष्ट प्रबंधन में प्रगति को बनाए रखना और <br> आगे बढ़ाना है</p>
-                            <div><a class="sb-btn-default mt-10" href="https://nulp.niua.org/resources?SBMRegistration">Sign In</a>
+                            <div>
+                                <!-- <a class="sb-btn-default mt-10" href="https://nulp.niua.org/resources?SBMRegistration">Sign In</a> -->
```

Comment:
@shreyas1434shinde Do not remove, show that button in disable format

Reply:
@komalm okay